### DISCUSSION TITLE
Switching to a new database after startup

### DISCUSSION THREAD

**Discussion by fmay** [prisma/prisma#24602] [Q&A]
I have an application manages projects, each of which has its own identically structured, Prisma backed database. 

I have searched high and low (clearly not far enough in either direction) for how to get Prisma to connect to another database or to add a new connection, but without success.

Would greatly appreciate any pointers.

**Comment by ludralph** [comment]
Hi @fmay 👋 

Thank you for raising this question.

Have you seen this section of our [documentation](https://www.prisma.io/docs/orm/reference/prisma-client-reference#programmatically-override-a-datasource-url) that allows you to programmatically override the datasource URL of your database connection string?
For example:
```js
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient({
  datasourceUrl: 'postgresql://johndoe:randompassword@localhost:5432/mydb',
})
```
This would allow you to swap out the connection string to the desired one at startup.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
